--- blt2.4z/src/bltNsUtil.h.orig	2002-07-18 08:35:32.000000000 +0200
+++ blt2.4z/src/bltNsUtil.h	2005-07-24 12:07:44.825339104 +0200
@@ -47,8 +47,10 @@
 #endif
 
 
+#if (TCL_VERSION_NUMBER < _VERSION(8,0,0)) 
 EXTERN Tcl_Command Tcl_FindCommand _ANSI_ARGS_((Tcl_Interp *interp,
 	char *name, Tcl_Namespace *nsPtr, int flags));
+#endif
 
 #define NS_SEARCH_NONE		(0)
 #define NS_SEARCH_CURRENT	(1<<0)
@@ -56,14 +58,17 @@
 #define NS_SEARCH_BOTH		(NS_SEARCH_GLOBAL | NS_SEARCH_CURRENT)
 
 
+#if (TCL_VERSION_NUMBER < _VERSION(8,5,0))
 /*
  * Namespace procedures not prototyped defined in Tcl.h 
  */
 EXTERN Tcl_Namespace *Tcl_GetCurrentNamespace _ANSI_ARGS_((Tcl_Interp *interp));
 
 EXTERN Tcl_Namespace *Tcl_GetGlobalNamespace _ANSI_ARGS_((Tcl_Interp *interp));
+#endif
 
 #if (TCL_MAJOR_VERSION >= 8)
+#  if (TCL_VERSION_NUMBER < _VERSION(8,5,0))
 EXTERN Tcl_Namespace *Tcl_CreateNamespace _ANSI_ARGS_((Tcl_Interp *interp,
 	char *name, ClientData clientData, Tcl_NamespaceDeleteProc *nsDelProc));
 
@@ -74,6 +79,7 @@
 
 EXTERN int Tcl_Export _ANSI_ARGS_((Tcl_Interp *interp, Tcl_Namespace *nsPtr,
 	char *name, int resetFlag));
+#  endif
 
 EXTERN Tcl_Var Tcl_FindNamespaceVar _ANSI_ARGS_((Tcl_Interp *interp, char *name,
 	Tcl_Namespace *contextNsPtr, int flags));
