--- blt2.4u/configure.in.orig	2003-06-17 20:39:16.000000000 +0200
+++ blt2.4u/configure.in	2003-06-17 20:41:39.000000000 +0200
@@ -1055,9 +1055,9 @@
   *-linux*)
     SHLIB_CFLAGS="-fPIC"
     SHLIB_LD="${CC}"
-    SHLIB_LD_FLAGS="-rdynamic -shared -Wl,-E -Wl,-soname,${libName}.so -Wl,-rpath,${LD_RUN_PATH}"
+    SHLIB_LD_FLAGS="-rdynamic -shared -Wl,-E -Wl,-soname,${libName}.so"
     
-    LDFLAGS="-Wl,-rpath,${LD_RUN_PATH}"
+    LDFLAGS=""
     EXTRA_LIB_SPECS="-ldl"
     ;;
   
