diff -Nru blt2.4u/Makefile.in blt2.4u.new/Makefile.in
--- blt2.4u/Makefile.in	Thu May 18 02:00:14 2000
+++ blt2.4u.new/Makefile.in	Fri Jan  5 08:48:53 2001
@@ -45,16 +45,14 @@
 
 install-dirs:
 	@for i in $(instdirs) ; do \
-	    if test ! -d $$i ; then \
-		echo " mkdir $$i" ; \
-		mkdir $$i ; \
-	    fi ; \
+		echo " mkdir -p $(DESTDIR)$$i" ; \
+		mkdir -p $(DESTDIR)$$i ; \
         done
 
 install-readme:
-	$(INSTALL_DATA) $(srcdir)/README $(scriptdir)
-	$(INSTALL_DATA) $(srcdir)/PROBLEMS $(scriptdir)
-	$(INSTALL_DATA) $(srcdir)/NEWS $(scriptdir)
+	$(INSTALL_DATA) $(srcdir)/README $(DESTDIR)$(scriptdir)
+	$(INSTALL_DATA) $(srcdir)/PROBLEMS $(DESTDIR)$(scriptdir)
+	$(INSTALL_DATA) $(srcdir)/NEWS $(DESTDIR)$(scriptdir)
 
 clean:
 	(cd src; $(MAKE) clean)
diff -Nru blt2.4u/demos/Makefile.in blt2.4u.new/demos/Makefile.in
--- blt2.4u/demos/Makefile.in	Fri Jun 23 02:40:05 2000
+++ blt2.4u.new/demos/Makefile.in	Fri Jan  5 08:46:46 2001
@@ -46,31 +46,31 @@
 
 inst-scripts: 
 	for i in $(srcdir)/scripts/*.tcl ; do \
-            $(INSTALL) $$i $(destdir)/scripts ; \
+            $(INSTALL) $$i $(DESTDIR)$(destdir)/scripts ; \
 	done 
 	for i in $(demos) ; do \
-            $(INSTALL) $(srcdir)/$$i $(destdir)/$$i ; \
+            $(INSTALL) $(srcdir)/$$i $(DESTDIR)$(destdir)/$$i ; \
 	done 
 
 inst-bitmaps: 
 	for i in $(srcdir)/bitmaps/*.xbm ; do \
-	    $(INSTALL_DATA) $$i $(destdir)/bitmaps ; \
+	    $(INSTALL_DATA) $$i $(DESTDIR)$(destdir)/bitmaps ; \
 	done 
 	for i in $(srcdir)/bitmaps/hand/*.xbm ; do \
-	    $(INSTALL_DATA) $$i $(destdir)/bitmaps/hand ; \
+	    $(INSTALL_DATA) $$i $(DESTDIR)$(destdir)/bitmaps/hand ; \
 	done 
 	for i in $(srcdir)/bitmaps/fish/*.xbm ; do \
-	    $(INSTALL_DATA) $$i $(destdir)/bitmaps/fish ; \
+	    $(INSTALL_DATA) $$i $(DESTDIR)$(destdir)/bitmaps/fish ; \
 	done 
 
 inst-images: 
 	for i in $(srcdir)/images/*.gif $(srcdir)/images/*.ps ; do \
-	    $(INSTALL_DATA) $$i $(destdir)/images ; \
+	    $(INSTALL_DATA) $$i $(DESTDIR)$(destdir)/images ; \
 	done 
 
 inst-dirs:
 	@for i in $(instdirs) ; do \
-	    if test -d "$$i" ; then : ; else mkdir "$$i" ; fi ; \
+	    mkdir -p "$(DESTDIR)$$i" ; \
         done
 
 clean:
diff -Nru blt2.4u/library/Makefile.in blt2.4u.new/library/Makefile.in
--- blt2.4u/library/Makefile.in	Fri May  5 06:49:36 2000
+++ blt2.4u.new/library/Makefile.in	Fri Jan  5 08:47:13 2001
@@ -32,21 +32,21 @@
 
 mkdirs:
 	@for i in $(instdirs) ; do \
-	    if test -d $$i ; then : ; else mkdir $$i ; fi ; \
+	    mkdir -p $(DESTDIR)$$i ; \
         done
 
 dragdrop: mkdirs
 	for i in $(ddFiles) ; do \
-	    $(INSTALL_DATA) $(srcdir)/dd_protocols/$$i $(scriptdir)/dd_protocols ; \
+	    $(INSTALL_DATA) $(srcdir)/dd_protocols/$$i $(DESTDIR)$(scriptdir)/dd_protocols ; \
 	done 
 
 files: mkdirs
 	for i in $(miscFiles) ; do \
-	    $(INSTALL_DATA) $(srcdir)/$$i $(scriptdir) ; \
+	    $(INSTALL_DATA) $(srcdir)/$$i $(DESTDIR)$(scriptdir) ; \
 	done 
 
 package: mkdirs pkgIndex.tcl
-	$(INSTALL_DATA) pkgIndex.tcl $(scriptdir)
+	$(INSTALL_DATA) pkgIndex.tcl $(DESTDIR)$(scriptdir)
 
 pkgIndex: 
 	rm -f pkgIndex.tcl
diff -Nru blt2.4u/man/Makefile.in blt2.4u.new/man/Makefile.in
--- blt2.4u/man/Makefile.in	Tue Feb  8 22:40:26 2000
+++ blt2.4u.new/man/Makefile.in	Fri Jan  5 08:47:44 2001
@@ -28,14 +28,12 @@
 
 install-manuals: $(MANPAGES)
 	for i in *.n ; do \
-		$(INSTALL_DATA) $$i $(sectiondir); \
+		$(INSTALL_DATA) $$i $(DESTDIR)$(sectiondir); \
 	done
 
 install-dirs:
 	@for i in $(instdirs) ; do \
-	    if test ! -d $$i ; then \
-		mkdir $$i ; \
-	    fi ; \
+		mkdir -p $(DESTDIR)$$i ; \
 	done
 
 .SUFFIXES: .n .man
diff -Nru blt2.4u/src/Makefile.in blt2.4u.new/src/Makefile.in
--- blt2.4u/src/Makefile.in	Fri May  5 06:49:36 2000
+++ blt2.4u.new/src/Makefile.in	Fri Jan  5 08:48:09 2001
@@ -143,30 +143,28 @@
 install: install_dirs install_lib install_demo install_headers
 
 install_demo: $(bltwish) $(bltsh)
-	$(INSTALL) -m 0755 $(bltwish) $(bindir)
-	$(INSTALL) -m 0755 $(bltsh) $(bindir)
+	$(INSTALL) -m 0755 $(bltwish) $(DESTDIR)$(bindir)
+	$(INSTALL) -m 0755 $(bltsh) $(DESTDIR)$(bindir)
 
 install_lib: $(lib_a)
-	$(INSTALL_DATA) $(lib_a) $(libdir)/$(libvers_a)
-	(cd $(libdir); $(RM) $(lib_a) ; $(LN_S) $(libvers_a) $(lib_a))
-	$(RANLIB) $(libdir)/$(libvers_a)
-	$(INSTALL_DATA) $(tcl_only_lib_a) $(libdir)/$(tcl_only_libvers_a)
-	(cd $(libdir); $(RM) $(tcl_only_lib_a) ; $(LN_S) $(tcl_only_libvers_a) $(tcl_only_lib_a))
-	$(RANLIB) $(libdir)/$(tcl_only_libvers_a)
+	$(INSTALL_DATA) $(lib_a) $(DESTDIR)$(libdir)/$(libvers_a)
+	(cd $(DESTDIR)$(libdir); $(RM) $(lib_a) ; $(LN_S) $(libvers_a) $(lib_a))
+	$(RANLIB) $(DESTDIR)$(libdir)/$(libvers_a)
+	$(INSTALL_DATA) $(tcl_only_lib_a) $(DESTDIR)$(libdir)/$(tcl_only_libvers_a)
+	(cd $(DESTDIR)$(libdir); $(RM) $(tcl_only_lib_a) ; $(LN_S) $(tcl_only_libvers_a) $(tcl_only_lib_a))
+	$(RANLIB) $(DESTDIR)$(libdir)/$(tcl_only_libvers_a)
 	(cd shared; $(MAKE) install) 
 
 install_dirs:
 	@for i in $(instdirs) ; do \
-	    if test ! -d $$i ; then \
-		echo " mkdir $$i" ; \
-		mkdir $$i ; \
-	    fi ; \
+		echo " mkdir -p $(DESTDIR)$$i" ; \
+		mkdir -p $(DESTDIR)$$i ; \
         done
 
 install_headers:
 	@for i in $(headers) ; do \
-	    echo "installing $(incdir)/$$i..." ; \
-	    $(INSTALL_DATA) -m 0444 $(srcdir)/$$i $(incdir) ; \
+	    echo "installing $(DESTDIR)$(incdir)/$$i..." ; \
+	    $(INSTALL_DATA) -m 0444 $(srcdir)/$$i $(DESTDIR)$(incdir) ; \
 	done
 
 lint:
diff -Nru blt2.4u/src/shared/Makefile.in blt2.4u.new/src/shared/Makefile.in
--- blt2.4u/src/shared/Makefile.in	Fri Jun 23 02:40:06 2000
+++ blt2.4u.new/src/shared/Makefile.in	Fri Jan  5 08:48:30 2001
@@ -124,19 +124,17 @@
 install: install_dirs install_lib install_demo
 
 install_demo: $(bltwish)
-	$(INSTALL) -m 0755 bltwish$(version) $(bindir)
-	$(INSTALL) -m 0755 bltsh$(version) $(bindir)
+	$(INSTALL) -m 0755 bltwish$(version) $(DESTDIR)$(bindir)
+	$(INSTALL) -m 0755 bltsh$(version) $(DESTDIR)$(bindir)
 
 install_lib: $(lib_so)
-	$(INSTALL) -m 0755 $(lib_so) $(libdir)
-	$(INSTALL) -m 0755 $(tcl_only_lib_so) $(libdir)
+	$(INSTALL) -m 0755 $(lib_so) $(DESTDIR)$(libdir)
+	$(INSTALL) -m 0755 $(tcl_only_lib_so) $(DESTDIR)$(libdir)
 
 install_dirs:
 	@for i in $(instdirs) ; do \
-	    if test ! -d $$i ; then \
-		echo " mkdir $$i" ; \
-		mkdir $$i ; \
-	    fi ; \
+		echo " mkdir $(DESTDIR)$$i" ; \
+		mkdir -p $(DESTDIR)$$i ; \
         done
 
 clean:
