--- blt2.4z/configure.in.orig	2005-07-23 19:10:46.000000000 +0200
+++ blt2.4z/configure.in	2005-07-23 19:12:48.000000000 +0200
@@ -188,8 +188,6 @@
 # Libraries:	Check for libraries used
 #
 # -----------------------------------------------------------------------
-AC_CHECK_LIB(socket, main)
-AC_CHECK_LIB(nsl, main)
 AC_CHECK_LIB(m, main)
 # -----------------------------------------------------------------------
 #
