diff -Nur blt2.4u.o/demos/barchart1.tcl blt2.4u/demos/barchart1.tcl
--- blt2.4u.o/demos/barchart1.tcl	Thu Oct 14 07:31:23 1999
+++ blt2.4u/demos/barchart1.tcl	Thu Jan  4 17:36:30 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/barchart2.tcl blt2.4u/demos/barchart2.tcl
--- blt2.4u.o/demos/barchart2.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/barchart2.tcl	Thu Jan  4 17:36:48 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/barchart3.tcl blt2.4u/demos/barchart3.tcl
--- blt2.4u.o/demos/barchart3.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/barchart3.tcl	Thu Jan  4 17:36:55 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/barchart4.tcl blt2.4u/demos/barchart4.tcl
--- blt2.4u.o/demos/barchart4.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/barchart4.tcl	Thu Jan  4 17:37:05 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/barchart5.tcl blt2.4u/demos/barchart5.tcl
--- blt2.4u.o/demos/barchart5.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/barchart5.tcl	Thu Jan  4 17:37:11 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/bgexec1.tcl blt2.4u/demos/bgexec1.tcl
--- blt2.4u.o/demos/bgexec1.tcl	Tue Mar 21 01:26:21 2000
+++ blt2.4u/demos/bgexec1.tcl	Thu Jan  4 17:37:18 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/bgexec2.tcl blt2.4u/demos/bgexec2.tcl
--- blt2.4u.o/demos/bgexec2.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/bgexec2.tcl	Thu Jan  4 17:37:25 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/bgexec3.tcl blt2.4u/demos/bgexec3.tcl
--- blt2.4u.o/demos/bgexec3.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/bgexec3.tcl	Thu Jan  4 17:37:31 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/bgexec4.tcl blt2.4u/demos/bgexec4.tcl
--- blt2.4u.o/demos/bgexec4.tcl	Wed Oct 13 05:55:00 1999
+++ blt2.4u/demos/bgexec4.tcl	Thu Jan  4 17:37:37 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/bgexec5.tcl blt2.4u/demos/bgexec5.tcl
--- blt2.4u.o/demos/bgexec5.tcl	Tue Mar 21 01:26:21 2000
+++ blt2.4u/demos/bgexec5.tcl	Thu Jan  4 17:37:43 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/bitmap.tcl blt2.4u/demos/bitmap.tcl
--- blt2.4u.o/demos/bitmap.tcl	Wed Oct 13 05:55:00 1999
+++ blt2.4u/demos/bitmap.tcl	Thu Jan  4 17:37:50 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/busy1.tcl blt2.4u/demos/busy1.tcl
--- blt2.4u.o/demos/busy1.tcl	Mon Mar 27 18:13:27 2000
+++ blt2.4u/demos/busy1.tcl	Thu Jan  4 17:37:57 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/busy2.tcl blt2.4u/demos/busy2.tcl
--- blt2.4u.o/demos/busy2.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/busy2.tcl	Thu Jan  4 17:38:05 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/container.tcl blt2.4u/demos/container.tcl
--- blt2.4u.o/demos/container.tcl	Tue Feb 29 21:48:00 2000
+++ blt2.4u/demos/container.tcl	Thu Jan  4 17:38:15 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 namespace import blt::*
diff -Nur blt2.4u.o/demos/dnd1.tcl blt2.4u/demos/dnd1.tcl
--- blt2.4u.o/demos/dnd1.tcl	Fri Jun 23 02:40:05 2000
+++ blt2.4u/demos/dnd1.tcl	Thu Jan  4 17:38:21 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 
diff -Nur blt2.4u.o/demos/dnd2.tcl blt2.4u/demos/dnd2.tcl
--- blt2.4u.o/demos/dnd2.tcl	Fri Jun 23 02:40:05 2000
+++ blt2.4u/demos/dnd2.tcl	Thu Jan  4 17:38:27 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/dragdrop1.tcl blt2.4u/demos/dragdrop1.tcl
--- blt2.4u.o/demos/dragdrop1.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/dragdrop1.tcl	Thu Jan  4 17:38:34 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 
diff -Nur blt2.4u.o/demos/dragdrop2.tcl blt2.4u/demos/dragdrop2.tcl
--- blt2.4u.o/demos/dragdrop2.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/dragdrop2.tcl	Thu Jan  4 17:38:40 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 
diff -Nur blt2.4u.o/demos/eps.tcl blt2.4u/demos/eps.tcl
--- blt2.4u.o/demos/eps.tcl	Wed Oct 13 05:55:00 1999
+++ blt2.4u/demos/eps.tcl	Thu Jan  4 17:38:47 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/graph1.tcl blt2.4u/demos/graph1.tcl
--- blt2.4u.o/demos/graph1.tcl	Tue Mar 21 01:26:21 2000
+++ blt2.4u/demos/graph1.tcl	Thu Jan  4 17:38:53 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 
diff -Nur blt2.4u.o/demos/graph2.tcl blt2.4u/demos/graph2.tcl
--- blt2.4u.o/demos/graph2.tcl	Fri Jun 23 02:40:05 2000
+++ blt2.4u/demos/graph2.tcl	Thu Jan  4 17:39:03 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/graph3.tcl blt2.4u/demos/graph3.tcl
--- blt2.4u.o/demos/graph3.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/graph3.tcl	Thu Jan  4 17:39:13 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/graph4.tcl blt2.4u/demos/graph4.tcl
--- blt2.4u.o/demos/graph4.tcl	Fri Jun 23 03:40:48 2000
+++ blt2.4u/demos/graph4.tcl	Thu Jan  4 17:39:19 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/graph5.tcl blt2.4u/demos/graph5.tcl
--- blt2.4u.o/demos/graph5.tcl	Mon Mar 27 18:13:27 2000
+++ blt2.4u/demos/graph5.tcl	Thu Jan  4 17:39:27 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/graph6.tcl blt2.4u/demos/graph6.tcl
--- blt2.4u.o/demos/graph6.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/graph6.tcl	Thu Jan  4 17:39:34 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/graph7.tcl blt2.4u/demos/graph7.tcl
--- blt2.4u.o/demos/graph7.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/graph7.tcl	Thu Jan  4 17:40:07 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 set blt_library ../library
 package require BLT
diff -Nur blt2.4u.o/demos/hierbox1.tcl blt2.4u/demos/hierbox1.tcl
--- blt2.4u.o/demos/hierbox1.tcl	Tue Mar 21 01:26:21 2000
+++ blt2.4u/demos/hierbox1.tcl	Thu Jan  4 17:40:14 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/hierbox2.tcl blt2.4u/demos/hierbox2.tcl
--- blt2.4u.o/demos/hierbox2.tcl	Tue Mar 21 01:26:21 2000
+++ blt2.4u/demos/hierbox2.tcl	Thu Jan  4 17:40:25 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/hierbox3.tcl blt2.4u/demos/hierbox3.tcl
--- blt2.4u.o/demos/hierbox3.tcl	Tue Mar 21 01:26:21 2000
+++ blt2.4u/demos/hierbox3.tcl	Thu Jan  4 17:40:33 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/hierbox4.tcl blt2.4u/demos/hierbox4.tcl
--- blt2.4u.o/demos/hierbox4.tcl	Tue Mar 21 01:26:21 2000
+++ blt2.4u/demos/hierbox4.tcl	Thu Jan  4 17:40:40 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/hiertable1.tcl blt2.4u/demos/hiertable1.tcl
--- blt2.4u.o/demos/hiertable1.tcl	Fri Jun 23 02:40:05 2000
+++ blt2.4u/demos/hiertable1.tcl	Thu Jan  4 17:40:48 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/hiertable2.tcl blt2.4u/demos/hiertable2.tcl
--- blt2.4u.o/demos/hiertable2.tcl	Fri Jun 23 03:40:48 2000
+++ blt2.4u/demos/hiertable2.tcl	Thu Jan  4 17:40:55 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/hiertable3.tcl blt2.4u/demos/hiertable3.tcl
--- blt2.4u.o/demos/hiertable3.tcl	Tue Jun 13 04:42:25 2000
+++ blt2.4u/demos/hiertable3.tcl	Thu Jan  4 17:41:07 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/htext1.tcl blt2.4u/demos/htext1.tcl
--- blt2.4u.o/demos/htext1.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/htext1.tcl	Thu Jan  4 17:41:26 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/scripts/page.tcl blt2.4u/demos/scripts/page.tcl
--- blt2.4u.o/demos/scripts/page.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/scripts/page.tcl	Thu Jan  4 17:35:21 2001
@@ -1,4 +1,4 @@
-#!/usr/local/bin/tclsh
+#!/usr/bin/tclsh
 
 array set page "
     rows	2
diff -Nur blt2.4u.o/demos/scripts/xcolors.tcl blt2.4u/demos/scripts/xcolors.tcl
--- blt2.4u.o/demos/scripts/xcolors.tcl	Fri May  5 06:49:36 2000
+++ blt2.4u/demos/scripts/xcolors.tcl	Thu Jan  4 17:36:19 2001
@@ -1,4 +1,4 @@
-#!../bltwish
+#!/usr/bin/bltwish
 #
 #  Tk version of xcolors
 #
diff -Nur blt2.4u.o/demos/spline.tcl blt2.4u/demos/spline.tcl
--- blt2.4u.o/demos/spline.tcl	Wed Jan 12 21:16:26 2000
+++ blt2.4u/demos/spline.tcl	Thu Jan  4 17:41:38 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/stripchart1.tcl blt2.4u/demos/stripchart1.tcl
--- blt2.4u.o/demos/stripchart1.tcl	Mon Mar 27 18:13:27 2000
+++ blt2.4u/demos/stripchart1.tcl	Thu Jan  4 17:41:46 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/tabnotebook1.tcl blt2.4u/demos/tabnotebook1.tcl
--- blt2.4u.o/demos/tabnotebook1.tcl	Fri May  5 06:49:36 2000
+++ blt2.4u/demos/tabnotebook1.tcl	Thu Jan  4 17:42:00 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/tabnotebook2.tcl blt2.4u/demos/tabnotebook2.tcl
--- blt2.4u.o/demos/tabnotebook2.tcl	Fri May  5 06:49:36 2000
+++ blt2.4u/demos/tabnotebook2.tcl	Thu Jan  4 17:42:06 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 
diff -Nur blt2.4u.o/demos/tabnotebook3.tcl blt2.4u/demos/tabnotebook3.tcl
--- blt2.4u.o/demos/tabnotebook3.tcl	Fri May  5 06:49:36 2000
+++ blt2.4u/demos/tabnotebook3.tcl	Thu Jan  4 17:42:17 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/tabset1.tcl blt2.4u/demos/tabset1.tcl
--- blt2.4u.o/demos/tabset1.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/tabset1.tcl	Thu Jan  4 17:42:24 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/tabset2.tcl blt2.4u/demos/tabset2.tcl
--- blt2.4u.o/demos/tabset2.tcl	Tue Jun 13 02:21:50 2000
+++ blt2.4u/demos/tabset2.tcl	Thu Jan  4 17:42:33 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 
diff -Nur blt2.4u.o/demos/tabset3.tcl blt2.4u/demos/tabset3.tcl
--- blt2.4u.o/demos/tabset3.tcl	Fri May  5 06:49:36 2000
+++ blt2.4u/demos/tabset3.tcl	Thu Jan  4 17:42:47 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/tabset4.tcl blt2.4u/demos/tabset4.tcl
--- blt2.4u.o/demos/tabset4.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/tabset4.tcl	Thu Jan  4 17:42:54 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/tour.tcl blt2.4u/demos/tour.tcl
--- blt2.4u.o/demos/tour.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/demos/tour.tcl	Thu Jan  4 17:43:01 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 #package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/demos/winop1.tcl blt2.4u/demos/winop1.tcl
--- blt2.4u.o/demos/winop1.tcl	Wed Oct 27 02:16:48 1999
+++ blt2.4u/demos/winop1.tcl	Thu Jan  4 17:43:13 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/examples/calendar.tcl blt2.4u/examples/calendar.tcl
--- blt2.4u.o/examples/calendar.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/examples/calendar.tcl	Thu Jan  4 17:43:22 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 
diff -Nur blt2.4u.o/examples/form.tcl blt2.4u/examples/form.tcl
--- blt2.4u.o/examples/form.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/examples/form.tcl	Thu Jan  4 17:43:28 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 # --------------------------------------------------------------------------
diff -Nur blt2.4u.o/examples/pareto.tcl blt2.4u/examples/pareto.tcl
--- blt2.4u.o/examples/pareto.tcl	Fri Oct  8 08:40:46 1999
+++ blt2.4u/examples/pareto.tcl	Thu Jan  4 17:43:36 2001
@@ -1,4 +1,4 @@
-#!../src/bltwish
+#!/usr/bin/bltwish
 
 package require BLT
 
